scripts/build/ci-build/Dockerfile

```diff
@@ -107,18 +108,16 @@ RUN apt-get update && \
         rubygems        \
         unzip &&        \
     gem install -N fpm && \
-    ln -s /usr/bin/llvm-dsymutil-6.0 /usr/bin/dsymutil                  && \
-    curl -fL https://nodejs.org/dist/v${NODEVERSION}/node-v${NODEVERSION}-linux-x64.tar.xz \
-      | tar -xJ --strip-components=1 -C /usr/local                      && \
-    curl -fsS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -   && \
-    echo "deb [arch=amd64] https://dl.yarnpkg.com/debian/ stable main"     \
-      | tee /etc/apt/sources.list.d/yarn.list                           && \
-    apt-get update && apt-get install -yq --no-install-recommends yarn      && \
+    ln -s /usr/bin/llvm-dsymutil-6.0 /usr/bin/dsymutil && \
+    curl -fsL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get update && apt-get install -yq nodejs=${NODEVERSION} && \
+    curl -fsS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    apt-get update && apt-get install -yq yarn=${YARNVERSION} && \
```

Comment:
I guess `--no-install-recommends` is removed intentionally?

Reply:
@papagian I opted to copy the [official](https://classic.yarnpkg.com/en/docs/install/#debian-stable) installation instruction: `sudo apt update && sudo apt install yarn`. I don't know how they design the package with regards to what "recommendations" get installed.

They do mention in the same document though that you can use `--no-install-recommends` if you don't want to install Node (and use e.g. nvm instead). Based on that, it sounds as if we should use their recommendations, in order to ensure that system dependencies are in place.